packages/squiggle-lang/src/dist/SymbolicDist.ts

```diff
@@ -576,13 +590,34 @@ export class Lognormal extends SymbolicDist {
     );
   }
 
-  static from90PercentCI(low: number, high: number) {
+  static fromCredibleInterval({
+    low,
+    high,
+    probability,
+  }: {
+    low: number;
+    high: number;
+    probability: number;
+  }): result<Lognormal, string> {
+    if (low >= high) {
+      return Result.Error("Low value must be less than high value");
+    }
+    if (low <= 0) {
+      return Result.Error("Low value must be above 0");
+    }
+    if (probability <= 0 || probability >= 1) {
+      return Result.Error("Probability must be in (0, 1) interval");
+    }
+
     const logLow = Math.log(low);
     const logHigh = Math.log(high);
-    const mu = E_A_Floats.mean([logLow, logHigh]);
-    const sigma = (logHigh - logLow) / (2 * normal95confidencePoint);
+
+    const normalizedSigmas = jstat.normal.inv(1 - (1 - probability) / 2, 0, 1);
+    const mu = (logLow + logHigh) / 2;
+    const sigma = (logHigh - logLow) / (2 * normalizedSigmas);
```

Comment:
Note: I could reduce the implementation to `Normal.fromCredibleInterval` call with logLow/logHigh, but that's not how it was done previously and I was lazy about it.

Maybe I should've, since there's some additional copy-paste here, especially with new arg checks.

Reply:
Let's leave for now.

On copy-paste, happy to follow the rule "refactor when there are three instanced"